recipe/meta.yaml

```diff
@@ -26,14 +26,18 @@ requirements:
   build:
     - toolchain
     - python
-    - numpy x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
     - vc 9              # [win and py27]
     - vc 14             # [win and py>=35]
     - boost-cpp {{ version }}
 
   run:
     - python
-    - numpy x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
```

Comment:
Only when the PR is ready. Not when I want to ensure that the same `numpy` is used to check the linkages and the `mkl` vs `openblas` situation.